galeracluster/source/glossary.rst

```diff
@@ -6,6 +6,13 @@
 .. glossary::
    :sorted:
 
+
+   Galera Arbitrator
+      If the expense of adding, for example, a third datacenter is too high, you can use the Galera arbitrator. An arbitrator is a member of the cluster which participates in voting, but not in actual replication.

```

Comment:
Garbd's do indeed participate in replication.  They don't generate any replication events of their own, nor do they store their replicated data, but replication can route through them and they must acknowledge all replication events just like any other galera node.